orchagent/crmorch.cpp

```diff
@@ -439,6 +439,10 @@ void CrmOrch::getResAvailableCounters()
         sai_attribute_t attr;
         attr.id = crmResSaiAvailAttrMap.at(res.first);
 
+        // ignore unsupported resources
+        if (res.second.resStatus != CrmResourceStatus::CRM_RES_SUPPORTED)
+            continue;
```

Comment:
@PrabhuSreenivasan please use `{ }` notation and move it at the beginning of the loop

Reply:
@nazariig , Thanks. I have made the suggested changes.

Reply:
@PrabhuSreenivasan NP